cmd/pd-server/main.go

```diff
@@ -46,8 +49,53 @@ import (
 )
 
 func main() {
+	var ctx context.Context
+	var cancel context.CancelFunc
+	var svr basicsvr.Server
```

Comment:
```suggestion
	var (
	    ctx       context.Context
		cancel context.CancelFunc
		svr       basicsvr.Server
	)
```

Reply:
Good suggestion! This should be a more golang conventional style.